changelog.json

```
@@ -1,5 +1,54 @@
 {
   "releases": {
+    "1.7.1": [
+      "[New] You can now choose to bring your changes with you to a new branch or stash them on the current branch when switching branches - #6107",
+      "[New] Rebase your current branch onto another branch using a guided flow - #5953",
+      "[New] Recently opened repositories appear at the top of the repository list - #7132",
```

This feels like it's missing that repositories are grouped by owner now, isn't it?

---

I think this was overlooked because it's been on beta for a while. Here's what we had back in `1.6.3-beta3` for it:

```
 "[Improved] Group and filter repositories by owner - #6923",
```

What about something like this to cover both?

```suggestion
      "[New] Repository list entries grouped by owner, and recent repositories listed at top - #6923 #7132",
```